tests/phpunit/class-wc-stripe-intent-controller-test.php

```
@@ -626,6 +626,59 @@ function ( $id ) use ( $subscription ) {
+		ob_start();
+		$controller->update_order_status_ajax();
+		$output   = ob_get_clean();
+		$response = json_decode( $output, true );
+
+		// Response must indicate failure so the frontend can show a message.
+		$this->assertFalse( $response['success'] );
+		$this->assertArrayHasKey( 'error', $response['data'] );
+		$this->assertStringContainsString( 'cancelled', strtolower( $response['data']['error']['message'] ) );
+
+		// Order must NOT be set to failed — the customer should be able to retry.
+		$final_order = wc_get_order( $order_id );
```

PHPMD (2.15.0)

[error] The variable $final_order is not named in camelCase. (undefined)

(CamelCaseVariableName)